ibm/Dockerfile.j2

```diff
@@ -0,0 +1,33 @@
+FROM python:{{ BASE_IMAGE_VERSION }}-slim-buster
+
+LABEL maintainer="Maxime Sibellas <maxime.sibellas@ekino.com>"
+
+ARG TERRAFORM_VERSION
+ARG IBM_PROVIDER_VERSION
+ARG IBM_CLI_VERSION
+
+# ENV PATH=$PATH:/usr/local/bin
+
+WORKDIR /builds
+
+RUN pip install -U pip \
+    && pip install awscli boto3 pipenv
+
+RUN apt-get update -qq \
+    && apt-get -y -qq install wget \
+    && echo "Install IBMCloud CLI" && \
+    wget https://public.dhe.ibm.com/cloud/bluemix/cli/bluemix-cli/${IBM_CLI_VERSION}/binaries/IBM_Cloud_CLI_${IBM_CLI_VERSION}_linux_amd64.tgz && \
+    tar -xzf IBM_Cloud_CLI_${IBM_CLI_VERSION}_linux_amd64.tgz && \
+    mv IBM_Cloud_CLI/ibmcloud /usr/local/bin && \
+    rm -rf IBM_Cloud_CLI_${IBM_CLI_VERSION}_linux_amd64.tgz IBM_Cloud_CLI \
+    && apt-get -y -qq remove wget && apt-get -y -qq autoremove
+RUN apt-get -y -qq install unzip wget \
+    && wget https://releases.hashicorp.com/terraform/${TERRAFORM_VERSION}/terraform_${TERRAFORM_VERSION}_linux_amd64.zip -O terraform.zip \
+    && unzip -o terraform.zip \
+    && rm terraform.zip \
+    && mv terraform /usr/bin/ \
```

Comment:
Why not `unzip terraform.zip terraform -d /usr/bin`, as is done for `linux_amd64.zip` below, instead of `unzip` + `mv`?

Reply:
You changed it to `unzip terraform.zip -d /usr/bin/`, but shouldn't we restrict what we extract from the zip, to be safe?

Reply:
The zipfile only contains terraform binary

Reply:
Right now, maybe, but what about in the future? You don't want to override a random binary in `/usr/bin`, especially if you can avoid it.

Reply:
Fixed for terraform and terraform provider for ibm